openlabcmd/requirements.txt

```diff
@@ -1,4 +1,5 @@
-pbr>=1.3
-PyYAML>=5.1
+enum;python_version=='2.7'
```

Comment:
Question: The update only is supported by py3,  or both py2 and py3? I find the default python version is python 2.7 in OpenLab CI host (zuul and nodepool), so we need to make sure openlabcmd can work on python 2.7

Reply:
openlabcmd support both py2 and py3.

`enum` in py3 is a built-in library, so we only install it in py2.

Reply:
OK, thank you